consensus/istanbul/core/core.go

```diff
@@ -289,6 +289,7 @@ func (c *core) startNewRound(round *big.Int) {
 	var roundChangeCertificate istanbul.RoundChangeCertificate
 	var request *istanbul.Request
 	if roundChange {
+		c.deletePrepareMessageFromDisk(c.current.Round())
```

Comment:
Might also have to delete old saved prepare messages when starting a new sequence.

Reply:
Changed to delete messages for a particular (sequence, round) pair.